src/Domain/GetSongsList.php

```diff
@@ -0,0 +1,10 @@
+<?php

```

Comment:
Data storage details belong to the Infrastructure not the Domain. Please move this file to `../Infrastructure` directory. Do the same for the test.